cves/CVE-2011-0698.yml

```diff
@@ -260,9 +296,13 @@ mistakes:
     they have written there. Are they doing those? Does the fix look proper?
 
     Use those questions to inspire your answer. Don't feel obligated to answer
-    every one. Write a thoughtful entry here that those ing the software
+    every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The specific mistakes are that the python std library was trusted without 
+    vetting to see if it met their specific requirements. Even a standard 
+    lib's functions must be vetted if you want to stay secure. Read the 
+    documentation, write positive and negative tests, stay aware.
```

Comment:
And even more important is to test the code. They should have tested both / and \